### ISSUE TITLE
Prompt of the Julia REPL is Main> instead of julia>

### ISSUE THREAD

**Issue by nbro** [JuliaEditorSupport/julia-vscode#479]
I set the value of `julia.executablePath` to be the result of `which julia`, which in my case is `/usr/local/bin/julia`. I also tried to set it to be `[Path to applications folder]/Julia-x.x.app/Contents/Resources/julia/bin/julia`. In both cases, though, I get a REPL with a prompt `Main>` instead of `julia>`. Why? How can I fix this?

I installed Julia using `brew cask`, specifically with the following command `brew cask install julia`.

**Comment by nbro** [comment]
I hadn't read the note for Mac OS X users, which says:

> When setting your julia.executablePath, you need to make sure that you are linking to the correct executable within your julia-x.x.app folder. The correct executable is located at

>     [Path to applications folder]/Julia-x.x.app/Contents/Resources/julia/bin/julia

I did that, I restarted VS Code, and now a REPL seems to open. However, its name is "Main" and not "Julia". Is this the expected behavior?

---

Btw, I had already tried to change the value of `julia.executablePath` in the configuration file to make it point to the result of `which julia`, which in my case is `/usr/local/bin/julia`, and I had obtained the same result, i.e. a REPL "Main", so I thought I wasn't suppose to make `julia.executablePath` point to `/usr/local/bin/julia`, given that I was expecting to see a REPL named "julia".
